src/main/java/org/candlepin/model/Pool.java

```diff
@@ -241,11 +244,11 @@ public String getDescription() {
     @NotNull
     private Date endDate;
 
-    @Column(name = "product_uuid", nullable = false)
+    @Column(name = "product_uuid", insertable = false, updatable = false, nullable = false)
```

Comment:
We should move the nullable flag to the product field we'll actually be maintaining

Reply:
Good point! Moved that back